Description: Add fcntl.h to winbindd_idmap2 to make it compile
Author: Luk Claes <luk@debian.org>
Origin: vendor
Bug: https://bugzilla.samba.org/show_bug.cgi?id=8091
Forwarded: https://bugzilla.samba.org/show_bug.cgi?id=8091

Index: experimental/source3/winbindd/idmap_tdb2.c
===================================================================
--- experimental.orig/source3/winbindd/idmap_tdb2.c
+++ experimental/source3/winbindd/idmap_tdb2.c
@@ -32,6 +32,7 @@
 */
 
 #include "includes.h"
+#include "system/filesys.h"
 #include "winbindd.h"
 #include "idmap.h"
 #include "idmap_rw.h"
